### PR TITLE
Add compact logging

### PR DESCRIPTION
**Pull request by RomainL972**
This was removed by the previous commits

### CODE REVIEW

**Review by robiot — APPROVED** [review]
Lgtm nice. Yep code getting removed is a problem. But we all have to 
cooperate, and don't merge before 2 people reviewed it.

**Comment by RomainL972** [comment]
Yes I agree. In this can I self approve or not ?

File: main.py [hunk 1]
```diff
@@ -32,21 +32,28 @@ def __init__(self):
...
         )
         self.proxies = (
             self.GetProxies(self.json_data["proxies"])
-            if "proxies" in self.json_data
+            if "proxies" in self.json_data and
+            self.json_data["proxies"] is not None
             else None
         )
+        self.compactlogging = (
+            self.json_data["compacy_logging"]
```
Comment (ixi2101) [inline main.py:52]: Should this  be compact_logging

**Review by ixi2101 — CHANGES_REQUESTED** [review]
self.json_data["compacy_logging"]  spelling

**Review by ixi2101 — APPROVED** [review]

**Pull request merged**